<launch>

    <node name="scout_odom" pkg="scout_base" type="scout_ros" output="screen">

    <param name="port_name" type="string" value="/dev/ttyUSB0" />
    <param name="baud_rate" value="115200" />

    <param name="odom_frame" type="string" value="odom" />
    <param name="base_frame" type="string" value="base_footprint" />

    <param name="control_rate" value="10" />

    </node>

</launch>
